### ISSUE TITLE
MathJax performance improvement

### ISSUE THREAD

**Issue by zsviczian** [zsviczian/obsidian-excalidraw-plugin#226]
Looking to load mathjax to an iframe

```javascript
const iframe = document.body.createEl("iframe");
iframe.title = "Excalidraw MathJax Support";
iframe.style.display = "none";
const doc = iframe.contentWindow.document;
const script = doc.createElement('script');
script.type = 'text/javascript';
script.onload = () => {
  const win = iframe.contentWindow;
  const tex = win.MathJax.tex2svg("\\sum_{a}^{b}\\frac{x}{2}",{display: true});
  const svg = tex.querySelector("svg");
  console.log(svg);
};
script.src = 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js';
doc.head.appendChild(script);
```

**Comment by zsviczian** [comment]
```javascript
f = app.vault.getAbstractFileByPath("Scripts/tex-svg.js");
s = await app.vault.read(f);
dataUrl = "data:text/javascript;base64," + btoa(unescape(encodeURIComponent(s)));

...
script.src = dataUrl;
```

**Comment by zsviczian** [comment]
released in 1.4.5
